tsconfig.build.json

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
This config isn't being used by the `build` command yet

Reply:
Oops! Fixed in [2f4bb6d](https://github.com/MetaMask/eth-block-tracker/pull/103/commits/2f4bb6d3e75d80459d607d719fe5670a11cb8bd9).